runelite-client/src/main/java/net/runelite/client/plugins/opponentinfo/OpponentInfoOverlay.java

```diff
@@ -176,4 +181,35 @@ else if (opponent instanceof Player)
 
 		return panelComponent.render(graphics);
 	}
+
+	private boolean bosHasHealthbar(int bossId)
+	{
+		switch (bossId)
+		{ //all defined in NpcID.java
```

Comment:
move this comment to new line, add space after  //

Reply:
Fixed

Reply:
Fixed